tests/cli/test_run.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
```

Comment:
why is this part of this PR?

Reply:
Same as test_rerun.py

Reply:
I don't follow - this is refactoring the run/rerun tests, so it seems to me that it should not be in this PR. What am I missing?

Reply:
Looks like github or somebody deleted my comment. :(

Test refactoring is on the todo list for a while now, and since I worked on workflow components as well here and during that refactored all this tests when removing this functionality from workflow commands (run/rerun/update). I decided to keep the tests refactored and sneak that refactoring in. If it is a problem I can create a ticket for this and refactor it on its own PR. I've seen no reason for all this extra work so I kept it like this since both of them are mergeable straight away.